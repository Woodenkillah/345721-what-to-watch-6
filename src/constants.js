export const DEFAULT_GENRE = `All genres`;

export const HttpStatusCodes = {
  BAD_REQUEST: 400,
  UNAUTHORIZED: 401
};

export const AuthErrorTypes = {
  BAD_REQUEST: `auth/loginOrPasswordError`,
  UNAUTHORIZED: `auth/accessDenied`
};

export const ReviewsErrorTypes = {
  BAD_REQUEST: `reviews/uploadReviewError`
};

export const LoadingStatuses = {
  PENDING: `load/pending`,
  LOADING: `load/loading`,
  LOADED: `load/loaded`,
  FAILED: `load/failed`
};

export const AuthStatuses = {
  AUTH: `auth/auth`,
  NO_AUTH: `auth/noAuth`
};

export const AppRoutes = {
  ROOT: `/`,
  LOGIN: `/login`,
  MY_LIST: `/mylist`,
  FILMS: `/films`,
  PLAYER: `/player`
};

export const ApiRoutes = {
  LOGIN: `/login`,
  LOGOUT: `/logout`,
  FILMS: `/films`,
  PROMO: `/films/promo`,
  COMMENTS: `/comments`,
  FAVORITES: `/favorite`
};

export const FavoriteStatuses = {
  ADD_FAVORITE: 1,
  REMOVE_FAVORITE: 0
};
