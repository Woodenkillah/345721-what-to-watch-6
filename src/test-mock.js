export const mockStringValue = `test`;

export const mockNumValue = 1;

export const mockBoolean = true;

export const mockStringSrc = `img/the-grand-budapest-hotel-poster.jpg`;

export const mockStrigArr = [`first`, `second`, `third`];

export const mockSimpleObj = {
  firstKey: `firstValue`,
  secondKey: `secondValue`
};

export const mockSimpleObjsArr = [
  {
    firstKey: `firstValue`,
    secondKey: `secondValue`
  },
  {
    firstKey: `firstValue`,
    secondKey: `secondValue`
  },
];

export const mockUserData = {
  email: `mock@gmail.ru`,
  password: `mockpass`
};

export const mockLoginUserData = {
  api: {
    "id": 1,
    "email": `Oliver.conner@gmail.com`,
    "name": `Oliver.conner`,
    "avatar_url": `img/1.png`
  },
  adapted: {
    email: `Oliver.conner@gmail.com`,
    avatar: `img/1.png`
  }
};

export const mockLogoutUserData = {
  email: ``,
  avatar: ``
};

export const mockUserReview = {
  id: 1,
  rating: 10,
  comment: `Awesome film!`
};

export const mockFilmReviewsData = {
  api: [
    {
      "id": 1,
      "user": {
        "id": 4,
        "name": `Kate Muir`
      },
      "rating": 8.9,
      "comment": `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
      "date": `2019-05-08T14:13:56.569Z`
    },
    {
      "id": 2,
      "user": {
        "id": 5,
        "name": `Kate Muir`
      },
      "rating": 8.9,
      "comment": `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
      "date": `2019-05-08T14:13:56.569Z`
    },
  ],
  adapted: [
    {
      id: 1,
      user: {
        id: 4,
        name: `Kate Muir`
      },
      rating: 8.9,
      comment: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
      date: `2019-05-08T14:13:56.569Z`
    },
    {
      id: 2,
      user: {
        id: 5,
        name: `Kate Muir`
      },
      rating: 8.9,
      comment: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
      date: `2019-05-08T14:13:56.569Z`
    }
  ]
};

export const mockSingleFilmData = {
  api: {
    "id": 2,
    "name": `Shutter Island`,
    "poster_image": `https://assets.htmlacademy.ru/intensives/javascript-3/film/poster/Shutter_Island.jpg`,
    "preview_image": `https://assets.htmlacademy.ru/intensives/javascript-3/film/preview/shutter-island.jpg`,
    "background_image": `https://assets.htmlacademy.ru/intensives/javascript-3/film/background/Shutter_Island.jpg`,
    "background_color": `#977461`,
    "video_link": `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    "preview_video_link": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "description": `In 1954, a U.S. Marshal investigates the disappearance of a murderer, who escaped from a hospital for the criminally insane.`,
    "rating": 4.1,
    "scores_count": 1002557,
    "director": `Martin Scorsese`,
    "starring": [`Leonardo DiCaprio`, `Emily Mortimer`, `Mark Ruffalo`],
    "run_time": 138,
    "genre": `Thriller`,
    "released": 2010,
    "is_favorite": false
  },
  adapted: {
    id: 2,
    name: `Shutter Island`,
    posterImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/poster/Shutter_Island.jpg`,
    previewImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/preview/shutter-island.jpg`,
    backgroundImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/background/Shutter_Island.jpg`,
    backgroundColor: `#977461`,
    videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    description: `In 1954, a U.S. Marshal investigates the disappearance of a murderer, who escaped from a hospital for the criminally insane.`,
    rating: 4.1,
    scoresCount: 1002557,
    director: `Martin Scorsese`,
    starring: [`Leonardo DiCaprio`, `Emily Mortimer`, `Mark Ruffalo`],
    runTime: 138,
    genre: `Thriller`,
    released: 2010,
    isFavorite: false
  }
};

export const mockFilmsListData = {
  api: [
    {
      "id": 1,
      "name": `The Grand Budapest Hotel`,
      "poster_image": `img/the-grand-budapest-hotel-poster.jpg`,
      "preview_image": `img/the-grand-budapest-hotel.jpg`,
      "background_image": `img/the-grand-budapest-hotel-bg.jpg`,
      "background_color": `#ffffff`,
      "video_link": `https://some-link`,
      "preview_video_link": `https://some-link`,
      "description": `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustave's friend and protege.`,
      "rating": 8.9,
      "scores_count": 240,
      "director": `Wes Andreson`,
      "starring": [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`, `Saoirse Ronan`],
      "run_time": 99,
      "genre": `Comedy`,
      "released": 2014,
      "is_favorite": false
    },
    {
      "id": 2,
      "name": `Shutter Island`,
      "poster_image": `https://assets.htmlacademy.ru/intensives/javascript-3/film/poster/Shutter_Island.jpg`,
      "preview_image": `https://assets.htmlacademy.ru/intensives/javascript-3/film/preview/shutter-island.jpg`,
      "background_image": `https://assets.htmlacademy.ru/intensives/javascript-3/film/background/Shutter_Island.jpg`,
      "background_color": `#977461`,
      "video_link": `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
      "preview_video_link": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
      "description": `In 1954, a U.S. Marshal investigates the disappearance of a murderer, who escaped from a hospital for the criminally insane.`,
      "rating": 4.1,
      "scores_count": 1002557,
      "director": `Martin Scorsese`,
      "starring": [`Leonardo DiCaprio`, `Emily Mortimer`, `Mark Ruffalo`],
      "run_time": 138,
      "genre": `Thriller`,
      "released": 2010,
      "is_favorite": false
    }
  ],
  adapted: [
    {
      id: 1,
      name: `The Grand Budapest Hotel`,
      posterImage: `img/the-grand-budapest-hotel-poster.jpg`,
      previewImage: `img/the-grand-budapest-hotel.jpg`,
      backgroundImage: `img/the-grand-budapest-hotel-bg.jpg`,
      backgroundColor: `#ffffff`,
      videoLink: `https://some-link`,
      previewVideoLink: `https://some-link`,
      description: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustave's friend and protege.`,
      rating: 8.9,
      scoresCount: 240,
      director: `Wes Andreson`,
      starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`, `Saoirse Ronan`],
      runTime: 99,
      genre: `Comedy`,
      released: 2014,
      isFavorite: false
    },
    {
      id: 2,
      name: `Shutter Island`,
      posterImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/poster/Shutter_Island.jpg`,
      previewImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/preview/shutter-island.jpg`,
      backgroundImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/background/Shutter_Island.jpg`,
      backgroundColor: `#977461`,
      videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
      previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
      description: `In 1954, a U.S. Marshal investigates the disappearance of a murderer, who escaped from a hospital for the criminally insane.`,
      rating: 4.1,
      scoresCount: 1002557,
      director: `Martin Scorsese`,
      starring: [`Leonardo DiCaprio`, `Emily Mortimer`, `Mark Ruffalo`],
      runTime: 138,
      genre: `Thriller`,
      released: 2010,
      isFavorite: false
    }
  ]
};

export const mockInitialStoreData = {
  auth: {
    authorizationStatus: `auth/noAuth`,
    user: {
      email: ``,
      avatar: ``
    },
    errorType: ``
  },
  targetFilm: {
    targetFilmData: {
      id: 0,
      name: ``,
      posterImage: ``,
      previewImage: ``,
      backgroundImage: ``,
      backgroundColor: ``,
      videoLink: ``,
      previewVideoLink: ``,
      description: ``,
      rating: 0,
      scoresCount: 0,
      director: ``,
      starring: [],
      runTime: 0,
      genre: ``,
      released: 0,
      isFavorite: false
    },
    films: {
      activeGenre: `All genres`,
      filmsData: [],
      filmsLoadingStatus: `load/loading`
    },
    promo: {
      promoData: {
        id: 0,
        name: ``,
        posterImage: ``,
        previewImage: ``,
        backgroundImage: ``,
        backgroundColor: ``,
        videoLink: ``,
        previewVideoLink: ``,
        description: ``,
        rating: 0,
        scoresCount: 0,
        director: ``,
        starring: [],
        runTime: 0,
        genre: ``,
        released: 0,
        isFavorite: false
      },
      promoLoadingStatus: `load/loading`
    },
    reviews: {
      reviewsList: []
    },
    favorites: {
      favoritesData: []
    }
  }
};

export const mockFullStoreData = {
  auth: {
    authorizationStatus: `auth/auth`,
    user: {
      email: `mock@gmail.ru`,
      avatar: `img/1.png`
    },
    errorType: ``
  },
  targetFilm: {
    targetFilmData: {
      id: 2,
      name: `Shutter Island`,
      posterImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/poster/Shutter_Island.jpg`,
      previewImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/preview/shutter-island.jpg`,
      backgroundImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/background/Shutter_Island.jpg`,
      backgroundColor: `#977461`,
      videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
      previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
      description: `In 1954, a U.S. Marshal investigates the disappearance of a murderer, who escaped from a hospital for the criminally insane.`,
      rating: 4.1,
      scoresCount: 1002557,
      director: `Martin Scorsese`,
      starring: [`Leonardo DiCaprio`, `Emily Mortimer`, `Mark Ruffalo`],
      runTime: 138,
      genre: `Thriller`,
      released: 2010,
      isFavorite: false
    },
    films: {
      activeGenre: `All genres`,
      filmsData: [
        {
          id: 1,
          name: `The Grand Budapest Hotel`,
          posterImage: `img/the-grand-budapest-hotel-poster.jpg`,
          previewImage: `img/the-grand-budapest-hotel.jpg`,
          backgroundImage: `img/the-grand-budapest-hotel-bg.jpg`,
          backgroundColor: `#ffffff`,
          videoLink: `https://some-link`,
          previewVideoLink: `https://some-link`,
          description: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustave's friend and protege.`,
          rating: 8.9,
          scoresCount: 240,
          director: `Wes Andreson`,
          starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`, `Saoirse Ronan`],
          runTime: 99,
          genre: `Comedy`,
          released: 2014,
          isFavorite: false
        },
        {
          id: 2,
          name: `Shutter Island`,
          posterImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/poster/Shutter_Island.jpg`,
          previewImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/preview/shutter-island.jpg`,
          backgroundImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/background/Shutter_Island.jpg`,
          backgroundColor: `#977461`,
          videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
          previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
          description: `In 1954, a U.S. Marshal investigates the disappearance of a murderer, who escaped from a hospital for the criminally insane.`,
          rating: 4.1,
          scoresCount: 1002557,
          director: `Martin Scorsese`,
          starring: [`Leonardo DiCaprio`, `Emily Mortimer`, `Mark Ruffalo`],
          runTime: 138,
          genre: `Thriller`,
          released: 2010,
          isFavorite: false
        }
      ],
      filmsLoadingStatus: `load/loaded`
    },
    promo: {
      promoData: {
        id: 2,
        name: `Shutter Island`,
        posterImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/poster/Shutter_Island.jpg`,
        previewImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/preview/shutter-island.jpg`,
        backgroundImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/background/Shutter_Island.jpg`,
        backgroundColor: `#977461`,
        videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
        previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
        description: `In 1954, a U.S. Marshal investigates the disappearance of a murderer, who escaped from a hospital for the criminally insane.`,
        rating: 4.1,
        scoresCount: 1002557,
        director: `Martin Scorsese`,
        starring: [`Leonardo DiCaprio`, `Emily Mortimer`, `Mark Ruffalo`],
        runTime: 138,
        genre: `Thriller`,
        released: 2010,
        isFavorite: false
      },
      promoLoadingStatus: `load/loaded`
    },
    reviews: {
      reviewsList: [
        {
          id: 1,
          user: {
            id: 4,
            name: `Kate Muir`
          },
          rating: 8.9,
          comment: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
          date: `2019-05-08T14:13:56.569Z`
        },
        {
          id: 2,
          user: {
            id: 5,
            name: `Kate Muir`
          },
          rating: 8.9,
          comment: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
          date: `2019-05-08T14:13:56.569Z`
        }
      ]
    },
    favorites: {
      favoritesData: [
        {
          id: 1,
          name: `The Grand Budapest Hotel`,
          posterImage: `img/the-grand-budapest-hotel-poster.jpg`,
          previewImage: `img/the-grand-budapest-hotel.jpg`,
          backgroundImage: `img/the-grand-budapest-hotel-bg.jpg`,
          backgroundColor: `#ffffff`,
          videoLink: `https://some-link`,
          previewVideoLink: `https://some-link`,
          description: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustave's friend and protege.`,
          rating: 8.9,
          scoresCount: 240,
          director: `Wes Andreson`,
          starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`, `Saoirse Ronan`],
          runTime: 99,
          genre: `Comedy`,
          released: 2014,
          isFavorite: false
        },
        {
          id: 2,
          name: `Shutter Island`,
          posterImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/poster/Shutter_Island.jpg`,
          previewImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/preview/shutter-island.jpg`,
          backgroundImage: `https://assets.htmlacademy.ru/intensives/javascript-3/film/background/Shutter_Island.jpg`,
          backgroundColor: `#977461`,
          videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
          previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
          description: `In 1954, a U.S. Marshal investigates the disappearance of a murderer, who escaped from a hospital for the criminally insane.`,
          rating: 4.1,
          scoresCount: 1002557,
          director: `Martin Scorsese`,
          starring: [`Leonardo DiCaprio`, `Emily Mortimer`, `Mark Ruffalo`],
          runTime: 138,
          genre: `Thriller`,
          released: 2010,
          isFavorite: false
        }
      ]
    }
  }
};

