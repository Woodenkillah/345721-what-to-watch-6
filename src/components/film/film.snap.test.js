import React from 'react';
import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';
import configureStore from 'redux-mock-store';
import * as redux from 'react-redux';
import Film from './film';
import {mockSingleFilmData, mockFilmsListData, mockInitialStoreData} from '../../test-mock';
import {AuthStatuses} from '../../constants';

const mockStore = configureStore({});

it(`Should render correectly`, () => {

  const history = createMemoryHistory();
  const store = mockStore(mockInitialStoreData);
  store.dispatch = () => {};
  const {container} = render(
      <redux.Provider store={store}>
        <Router history={history}>
          <Film
            filmsData={mockFilmsListData.adapted}
            targetFilmData={mockSingleFilmData.adapted}
            onSetFavorite={jest.fn()}
            authorizationStatus={AuthStatuses.AUTH}
            onFetchFilm={jest.fn()}
          />
        </Router>
      </redux.Provider>
  );

  expect(container).toMatchSnapshot();
});
