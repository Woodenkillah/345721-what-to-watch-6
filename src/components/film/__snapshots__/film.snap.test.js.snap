// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correectly 1`] = `
<div>
  <section
    class="movie-card movie-card--full"
  >
    <div
      class="movie-card__hero"
    >
      <div
        class="movie-card__bg"
      >
        <img
          alt=""
          src=""
        />
      </div>
      <h1
        class="visually-hidden"
      >
        WTW
      </h1>
      <header
        class="page-header movie-card__head"
      >
        <div
          class="logo"
        >
          <a
            aria-current="page"
            class="logo__link active"
            href="/"
          >
            <span
              class="logo__letter logo__letter--1"
            >
              W
            </span>
            <span
              class="logo__letter logo__letter--2"
            >
              T
            </span>
            <span
              class="logo__letter logo__letter--3"
            >
              W
            </span>
          </a>
        </div>
        <a
          href="/login"
          style="color: rgb(217, 205, 141); text-align: right; text-decoration: none; margin-left: auto; padding: 5px; border: 1px solid #d9cd8d; border-radius: 10px;"
        >
          <h2>
            Sign in
          </h2>
        </a>
      </header>
      <div
        class="movie-card__wrap"
      >
        <div
          class="movie-card__desc"
        >
          <h2
            class="movie-card__title"
          />
          <p
            class="movie-card__meta"
          >
            <span
              class="movie-card__genre"
            />
            <span
              class="movie-card__year"
            >
              0
            </span>
          </p>
          <div
            class="movie-card__buttons"
          >
            <button
              class="btn btn--play movie-card__button"
              type="button"
            >
              <svg
                height="19"
                viewBox="0 0 19 19"
                width="19"
              >
                <use
                  xlink:href="#play-s"
                />
              </svg>
              <span>
                Play
              </span>
            </button>
            <button
              class="btn btn--list movie-card__button"
              style="display: none;"
              type="button"
            >
              <svg
                height="20"
                viewBox="0 0 19 20"
                width="19"
              >
                <use
                  xlink:href="#add"
                />
              </svg>
              <span>
                My list
              </span>
            </button>
            <a
              class="btn movie-card__button"
              href="/films/NaN/review"
              style="display: none;"
            >
              Add review
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      class="movie-card__wrap movie-card__translate-top"
    >
      <div
        class="movie-card__info"
      >
        <div
          class="movie-card__poster movie-card__poster--big"
        >
          <img
            alt=""
            height="327"
            src=""
            width="218"
          />
        </div>
        <div
          class="movie-card__desc"
        >
          <nav
            class="movie-nav movie-card__nav"
          >
            <ul
              class="movie-nav__list"
            >
              <li
                class="movie-nav__item movie-nav__item--active"
              >
                <a
                  class="movie-nav__link"
                  href="#"
                >
                  Overview
                </a>
              </li>
              <li
                class="movie-nav__item"
              >
                <a
                  class="movie-nav__link"
                  href="#"
                >
                  Details
                </a>
              </li>
              <li
                class="movie-nav__item"
              >
                <a
                  class="movie-nav__link"
                  href="#"
                >
                  Reviews
                </a>
              </li>
            </ul>
          </nav>
          <div
            class="movie-rating"
          >
            <div
              class="movie-rating__score"
            >
              0
            </div>
            <p
              class="movie-rating__meta"
            >
              <span
                class="movie-rating__level"
              >
                Bad
              </span>
              <span
                class="movie-rating__count"
              >
                0
              </span>
            </p>
          </div>
          <div
            class="movie-card__text"
          >
            <p />
            <p
              class="movie-card__director"
            >
              <strong>
                Director: 
                
              </strong>
            </p>
            <p
              class="movie-card__starring"
            >
              <strong>
                Starring: 
              </strong>
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
  <div
    class="page-content"
  >
    <section
      class="catalog catalog--like-this"
    >
      <h2
        class="catalog__title"
      >
        More like this
      </h2>
      <div
        class="catalog__movies-list"
      >
        <h2>
          The were no films added yet.
        </h2>
      </div>
    </section>
    <footer
      class="page-footer"
    >
      <div
        class="logo"
      >
        <a
          class="logo__link logo__link--light"
        >
          <span
            class="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            class="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            class="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <div
        class="copyright"
      >
        <p>
          Â© 2021 What to watch Ltd.
        </p>
      </div>
    </footer>
  </div>
</div>
`;
